internal/store/bundle.go

```diff
@@ -97,6 +98,20 @@ func (b *bundleStore) List() ([]reference.Named, error) {
 	return references, nil
 }
 
+// Remove removes a bundle from the bundle store.
+func (b *bundleStore) Remove(ref reference.Named) error {
+	path, err := b.storePath(ref)
+	if err != nil {
+		return err
+	}
+
+	if _, err := os.Stat(path); os.IsNotExist(err) {
```

Comment:
What if the error is something else? Shouldn't we return either?

Reply:
If the error is something else (like can't stat because of permissions) then the remove will error out too, but here at least we have a nice message saying the thing is not there.